### PR TITLE
Edit welcome text

### LINKED ISSUE #3208: Delete "Release Notes" from header
image

### LINKED ISSUE #3209: Improve tutorial description
1. Remove "Tutorial" from header
2. Remove "Read the tutorial" Start with Learn...

image

### PR DESCRIPTION
**Pull request by elharo**
@chanseokoh @briandealwis 

fix #3208 
fix #3209

### CODE REVIEW

**Comment by elharo** [comment]
@briandealwis Do you know how the icons are linked here?

image

**Comment by chanseokoh** [comment]
> @briandealwis Do you know how the icons are linked here?

https://github.com/GoogleCloudPlatform/google-cloud-eclipse/blob/v1.7.1C/plugins/com.google.cloud.tools.eclipse.welcome/intro/css/intro.css#L5

File: plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml [hunk 1]
```diff
@@ -12,23 +12,35 @@
...
 
   <extensionContent path="tutorials/@" style="css/intro.css">
-    <group style-id="content-group">
+    <group style-id="content-group" label="Google Cloud Tools for Eclipse">
       <link id="cloud-tools-for-eclipse-tutorial-link"
-            label="Google Cloud Tools for Eclipse Tutorial"
+            label="App Engine Standard Environment"
+            style-id="content-link"
+            url="https://cloud.google.com/eclipse/docs/creating-new-webapp">
+        <text>Learn how to build, run, debug, and deploy App Engine standard environment applications with Eclipse.</text>
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml]: I see you removed the period for the release notes item. Remove the period to be consistent?

File: plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml [hunk 2]
```diff
@@ -12,23 +12,35 @@
...
+      </link>
+      <link id="cloud-tools-for-eclipse-flexible-link"
+            label="App Engine Flexible Environment"
             style-id="content-link"
-            url="https://cloud.google.com/eclipse/docs/quickstart">
-        <text>Read the Google Cloud Tools for Eclipse tutorial to learn how to build App Engine applications with Eclipse.</text>
+            url="https://cloud.google.com/eclipse/docs/create-flex-app">
+        <text>Learn how to build, run, debug, and deploy App Engine flexible environment applications with Eclipse.</text>
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml]: I see you removed the period for the release notes item. Remove the period to be consistent?

File: plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml (hunk 1 above)
Comment (elharo) [reply plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml]: done

File: plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml (hunk 2 above)
Comment (elharo) [reply plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml]: done

**Comment by chanseokoh** [comment]
Skip the icons for the flex and Dataflow entries? See

https://github.com/GoogleCloudPlatform/google-cloud-eclipse/blob/v1.7.1C/plugins/com.google.cloud.tools.eclipse.welcome/intro/css/intro.css#L5

**Pull request merged**